src/components/Button/content.js

```diff
@@ -16,12 +16,17 @@ const variantMap = {
     'border-filled': 'base',
 };
 
-export default function Content({ label, children, variant, isLoading }) {
+export default function Content({ label, children, variant, isLoading, size }) {
+    const spinnerMap = { small: 'x-small', medium: 'small', large: 'medium' };
```

Comment:
this can be outside, remember it creates an unnecessary new object each rerender